## Week 6 Hypothesis Test

### I chose BrainBoost Gum Co:

##### Chewing our nootropic gum for 10 minutes before an exam improves memory recall scores more than chewing regular mint gum.
##### Measure: memory test score (continuous)

#### Hasan Al-Quaid

### 1. What are your null and alternative hypotheses?

**Research Question**: Does chewing nootropic gum for 10 minutes before an exam improve memory recall scores more than chewing regular mint gum?

**Null Hypothesis ($H_0$)**: There is no difference in average memory recall scores between the nootropic gum group and the regular mint gum group.

**Alternative Hypothesis ($H_1$)**: The nootropic gum group has higher average memory recall scores than the regular mint gum group.

### 2. What statistical test will you perform?

I will use a **Independent samples t-test(one-sided)**.

**Rationale:**
- We are comparing mean memory recall scores, treated as a continuous outcome, between two independent groups: nootropic gum and regular mint gum.
- We are specifically interested in whether the nootropic gum group has higher memory recall scores.
- A one-sided test matches this directional hypothesis, which will be chosen before examining the results.

**Assumptions to check:**
- Approximate normality of score distributions within each group.
- Similar variances between groups(Will use Welch’s t-test if variances differ).
- Independence of observations: each participant contributes one score, and participants complete the test individually without influencing one another.

### 3. Explore literature to find effect size

I will use **Cohen’s d** because I am comparing mean memory
recall scores between two groups.


**Literature:**
    Onyper et al. (2011), Cognitive advantages of chewing gum. Now you see them, now you don’t. https://www.sciencedirect.com/science/article/pii/S0195666311004703?via%3Dihub

- Summary: They compared groups who chewed gum before or during testing with groups who did not chew gum. Chewing before testing improved some memory results, with benefits lasting about 15–20 minutes.

**Findings from the Literature:**

I selected the full-attention recall task from Experiment 1a
because participants learned words without a competing task
after chewing gum for 5 minutes. Their memory recall scores
were compared with those of a group that did not chew gum.

Table 1 reports:
| | Gum group | No-gum group |
|---|---:|---:|
| Mean recall score | 8.18 | 5.20 |
| Standard deviation | 3.19 | 2.30 |
| Participants | 40 | 40 |


**Calculations:**
1. Difference in means: 8.18 - 5.20 = 2.98
2. Calculate pooled standard deviation:
$$s_p = \sqrt{\frac{(n_1 - 1)s_1^2 + (n_2 - 1)s_2^2}{n_1 + n_2 - 2}}$$
$$s_p = \sqrt{\frac{(40 - 1)(3.19)^2 + (40 - 1)(2.30)^2}{40 + 40 - 2}} = \sqrt{\frac{603.1779}{78}} \approx 2.79$$


**Cohen’s d:**

d = Mean difference / Pooled standard deviation
- $d = 2.98 / 2.79 \approx 1.07$ (large effect)

But to be conservative, we will plan to for a medium effect size, d = 0.5

### 4. Perform power analysis w/ effect size

Determine the minimum sample size per group needed to detect an effect size of 0.5 with:
* Significance level ($\alpha$): 0.05 (one-sided)
* Power ($1 - \beta$): 0.80

In [16]:
from statsmodels.stats.power import TTestIndPower
import math

# Parameters
effect_size = 0.5
alpha = 0.05
power = 0.80

# Power analysis
analysis = TTestIndPower()
sample_size = analysis.solve_power(effect_size=effect_size, alpha=alpha, power=power, alternative='larger')

print(f"We would need {math.ceil(sample_size)} per group, and a total of {math.ceil(sample_size) * 2} participants")

We would need 51 per group, and a total of 102 participants


**Rational:**

With our sample size being 51, this tells us that we need 51 participants per group, with a total of 102 participants to detect an effect.

I selected a significance level of 0.05 and power of 0.80 because they are commonly used standards.